src/lib/viewers/image/ImageViewer.js

```diff
@@ -126,12 +126,26 @@ class ImageViewer extends ImageBaseViewer {
     zoom(type) {
         let newWidth;
         let newHeight;
-        const imageCurrentDimensions = this.imageEl.getBoundingClientRect(); // Getting bounding rect does not ignore transforms / rotates
+        let imageCurrentDimensions;
+        if (type === 'reset') {
```

Comment:
Can we keep this in the switch statement? You can set `imageCurrentDimensions` here and then update it in the switch statement if we are resetting. It feels a little weird to check for the 'reset' case and then switch on type right after.